.github/workflows/ci_nightly.yml

```diff
@@ -25,13 +26,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        device: [
-            # RISC-V devices:
-            { soc: "esp32c2", target: "riscv32imc-unknown-none-elf" },
-            { soc: "esp32c3", target: "riscv32imc-unknown-none-elf" },
-            { soc: "esp32c6", target: "riscv32imac-unknown-none-elf" },
-            { soc: "esp32h2", target: "riscv32imac-unknown-none-elf" },
-          ]
```

Comment:
The target wasnt used as we install both targets unconditionally, so I decided to remove it and simplify the mtrix